specifyweb/frontend/js_src/lib/components/FormEditor/spec.ts

```diff
@@ -48,6 +48,127 @@ export type ViewSets = Omit<RawViewSets, 'viewDefs'> & {
   >;
 };
 
+const tablesWithNoBusRulesIn6 = new Set([
```

Comment:
this list is rather long - probably longer than the inverse list
and also, it's unlikely that more business rules will be added in sp6, but it's highly likely that more tables will be added in sp7
thus, you should invert this list and define the tables with business rules